File: src/sdk-reference/cpp/1/auth/validate-my-credentials/snippets/validate-my-credentials.cpp
try {
  kuzzle->auth->login("local", "{\"username\":\"foo\",\"password\":\"bar\"}");
  bool valid = kuzzle->auth->validateMyCredentials("local", "{\"username\":\"foo\",\"password\":\"bar\"}");

  if (valid) {
    std::cout << "Credentials are valid" << std::endl;
  }
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/auth/validate-my-credentials/snippets/validate-my-credentials.cs
try {
  kuzzle.auth.login("local", @"{""username"":""foo"",""password"":""bar""}");
  bool valid = kuzzle.auth.validateMyCredentials("local", @"{""username"":""foo"",""password"":""bar""}");

  if (valid) {
    Console.WriteLine("Credentials are valid");
  }
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.Message());
}
